test/0.8.9/sepolia-deposit-adapter.test.js

```diff
@@ -0,0 +1,157 @@
+const { contract, artifacts, ethers } = require('hardhat')
+const { assert } = require('../helpers/assert')
+const { ETH } = require('../helpers/utils')
+
+const { EvmSnapshot } = require('../helpers/blockchain')
+
+const SepoliaDepositAdapter = artifacts.require('SepoliaDepositAdapter')
+const SepoliaDepositContract = artifacts.require('ISepoliaDepositContract')
+
+// To run Sepolia Deposit Adapter tests:
+// HARDHAT_FORKING_URL=<rpc url> HARDHAT_CHAIN_ID=11155111 npx hardhat test --grep "SepoliaDepositAdapter"
+contract('SepoliaDepositAdapter', ([deployer]) => {
+  let depositAdapter
+  let snapshot
+  let bepoliaToken
+  const sepoliaDepositContractAddress = '0x7f02C3E3c98b133055B8B348B2Ac625669Ed295D'
+  const EOAddress = '0x6885E36BFcb68CB383DfE90023a462C03BCB2AE5'
+  const bepoliaTokenHolder = EOAddress
+  // const log = console.log
+  const log = () => {}
+
+  before('deploy lido with dao', async function () {
+    const { chainId } = await ethers.provider.getNetwork()
+    if (chainId !== 11155111) {
+      return this.skip()
+    }
+
+    depositAdapter = await SepoliaDepositAdapter.new(sepoliaDepositContractAddress)
+    log('depositAdapter address', depositAdapter.address)
+
+    bepoliaToken = await ethers.getContractAt('ISepoliaDepositContract', sepoliaDepositContractAddress)
+
+    const code = await ethers.provider.getCode(depositAdapter.address)
+    assert.notEqual(code, '0x')
+
+    snapshot = new EvmSnapshot(ethers.provider)
+    await snapshot.make()
+  })
+
+  afterEach(async () => {
+    await snapshot.rollback()
+  })
+
+  describe('SepoliaDepositAdapter Logic', () => {
```

Comment:
thank you for improving the tests :+1: 

it would be great to test revert conditions if possible; though not a blocker for the testnet contract, ofc